### DISCUSSION TITLE
Prohibit line breaks

### DISCUSSION THREAD

**Discussion by Natureshadow** [typst/typst#1137] [Q&A] [answered]
What is the preferred way of prohibiting a line break between two words, or within a "span" of words?

LaTeX has `\~` for a non-breaking space, and of course there would be the Unicode codepoint for a non-breaking space, but…

**Comment by Natureshadow** [comment, marked as answer]
Ah, the short-hand symbol `~` works. It didn't because I was using a `string` instead of `content`.

**Comment by Enivex** [comment]
Non breaking space is simply ~, though you can still get line break in the middle of words. To prevent that you can wrap what you want to be on a single line in a #box()
